frontend/e2e/src/app.e2e-spec.ts

```diff
@@ -1,23 +1 @@
-import { AppPage } from './app.po';
-import { browser, logging } from 'protractor';
 
-describe('workspace-project App', () => {
```

Comment:
Why has this test ([and its helper](https://github.com/TilBlechschmidt/NAK-IAA/blob/fa0ba98bce3297d2ab7a1a8f36a86bf8b21ad658/frontend/e2e/src/app.po.ts)) been removed? It is the only E2E test which checks if the app is rendering and setting the correct `title` in the HTML `<head>`.